packages/auth/src/grant/routes.ts

```diff
@@ -96,7 +96,13 @@ async function createGrant(
   deps: ServiceDependencies,
   ctx: CreateContext
 ): Promise<void> {
-  if (canSkipInteraction(deps.config, ctx.request.body)) {
+  let noInteractionRequired: boolean
+  try {
+    noInteractionRequired = canSkipInteraction(deps.config, ctx.request.body)
+  } catch (err) {
+    ctx.throw(400, 'identifier_required', { error: 'identifier_required' })
```

Comment:
Note to self, or some other enterprising person: we should make it a point to change all the calls to `ctx.throw` so that they also use this pattern.